.nvmrc

```diff
@@ -1 +0,0 @@
-0.10.40
```

Comment:
but it requires to run `nvm install` anyway?  I'll add it back, even if we have to remember to keep it up to date